cmd/root.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	t "github.com/AirHelp/filler/templates"
+	"github.com/spf13/cobra"
+)
+
+const (
+	directoryToScanArgument = "dir"
+	fileExtensionArgument   = "ext"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "filler",
+	Short: "Filler - fill templates with environment variables",
+	Long:  "Filler - fill templates with environment variables",
+	RunE:  template,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.PersistentFlags().String(directoryToScanArgument, "", "directory where app will search for templates")
+	rootCmd.PersistentFlags().String(fileExtensionArgument, "", "template file extension")
```

Comment:
can we set a default `ext` value, for example, `tpl`?

Reply:
added